templates/solutions/index.html

```diff
@@ -12,6 +12,8 @@
   Discover Canonical's open source solutions. Cost-effective, composable, and completely under your control. From the OS to AI applications and MLOps.
 {% endblock %}
 
+{% block meta_image %}https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto,fl_sanitize,w_2748,h_1146/https://assets.ubuntu.com/v1/c55ce9f5-ubuntu%20platform.png{% endblock %}
```

Comment:
We don't explicitly declare the cloudinary version, that's done via the image template, but I've never seen us use the image template for meta images so I think we can just use the regular asset link
```suggestion
{% block meta_image %}https://assets.ubuntu.com/v1/c55ce9f5-ubuntu%20platform.png{% endblock %}
```